Podcast/UserSettings.swift

```diff
@@ -81,8 +81,18 @@ class UserSettings: NSObject {
             SettingsSection(id: "profile_settings", items: [
                 SettingsField(id: "username_disclosure", title: "Change Username", type: .disclosure, tapAction: {
                     settingsViewController.navigationController?.pushViewController(UserSettings.changeUsernamePage, animated: true)
+                }),
+            ]),
+            SettingsSection(id: "logout", items: [
+                SettingsField(id: "logout", title: "Logout", type: .button, tapAction: {
+                    guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+                    //TODO: add custom alert view in the future
+                    let alert = UIAlertController(title: "Log out", message: "Are you sure you want to logout?", preferredStyle: .alert)
```

Comment:
all of the logout related text should be all either "log out" or "logout"

Reply:
http://whitelightconcepts.com/login-vs-log-in-logout-vs-log-out/
We should probably use log out for both here